src/nvim/ex_cmds_defs.h

```diff
@@ -164,17 +164,17 @@ typedef struct expand {
 /// flag.  This needs to be saved for recursive commands, put them in a
 /// structure for easy manipulation.
 typedef struct {
-  int hide;          ///< TRUE when ":hide" was used
   int split;         ///< flags for win_split()
   int tab;           ///< > 0 when ":tab" was used
-  int confirm;       ///< TRUE to invoke yes/no dialog
-  int keepalt;       ///< TRUE when ":keepalt" was used
-  int keepmarks;     ///< TRUE when ":keepmarks" was used
-  int keepjumps;     ///< TRUE when ":keepjumps" was used
-  int lockmarks;     ///< TRUE when ":lockmarks" was used
-  int keeppatterns;  ///< TRUE when ":keeppatterns" was used
-  bool noswapfile;   ///< true when ":noswapfile" was used
   bool browse;       ///< TRUE to invoke file dialog
+  bool confirm;      ///< TRUE to invoke yes/no dialog
+  bool hide;         ///< TRUE when ":hide" was used
+  bool keepalt;      ///< TRUE when ":keepalt" was used
+  bool keepjumps;    ///< TRUE when ":keepjumps" was used
+  bool keepmarks;    ///< TRUE when ":keepmarks" was used
+  bool keeppatterns; ///< TRUE when ":keeppatterns" was used
+  bool lockmarks;    ///< TRUE when ":lockmarks" was used
```

Comment:
We use `true` instead of `TRUE` when the type of the variable is `bool`. Also in comments.